### PR TITLE
Feat: Home page

### PR DESCRIPTION
**Pull request by sri-rishi**
### Folders to check 

- page folder
- component folder
- context folder

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import "./App.css";
+import { Home } from "./pages/Home/Home";
 
 function App() {
   return (
     <div className="App">
-      
+      <Home />
```
Comment (bhtibrewal) [inline src/App.js:7]: This Files name should be App.jsx and not App.js

File: src/Components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { Button } from "../index";
+import { SearchBar } from "./SearchBar";
+
+export const Navbar = () => {
+    return (
+        <div>
+            <nav className="nav-box navbar-box flex-row align-center justify-evenly">
+                <div className="nav-section title-section flex-row align-center justify-evenly">
+                    <Button className="hamburger display-none btn-border-none bg-transparent" icon={<GiHamburgerMenu className="icon-vr-align"/>} />
+                    <a href="#">
```
Comment (bhtibrewal) [inline src/Components/Navbar/Navbar.jsx:11]: why are using anchor tag here

File: src/Components/VideoCard/VideoCard.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        // Nine Zeros for Billions
+        return Math.abs(Number(views)) >= 1.0e+9 
+        ? (Math.abs(Number(views)) / 1.0e+9).toFixed(1) + "B"
+        // Six Zeroes for Millions 
+        : Math.abs(Number(views)) >= 1.0e+6
+        ?( Math.abs(Number(views)) / 1.0e+6).toFixed(1) + "M"
+        // Three Zeroes for Thousands
+        : (Math.abs(Number(views))) >= 1.0e+3
+        ? (Math.abs(Number(views)) / 1.0e+3).toFixed(1) + "K"
+        : Math.abs(Number(views));
```
Comment (bhtibrewal) [inline src/Components/VideoCard/VideoCard.jsx:27]: Appreciate the thought here, but why haven't you saved `Math.abs(Number(views))` in a variable and then use it

File: src/Context/data.context.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import axios from "axios";
+import {createContext, useState, useContext, useEffect} from "react";
+const DataContext =  createContext();
+
+const DataProvider = ({children}) => {
+    const [videosList, setVideosList] = useState([]);
+    const [categoryList, setCategoryList] = useState([]);
+
+    useEffect(() => {
+        getData(setVideosList);
```
Comment (bhtibrewal) [inline src/Context/data.context.js:10]: the function name should be getVideoList instead

File: src/Context/data.context.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        getCategoryData(setCategoryList);
+    }, []);
+
+    const getCategoryData = async(setCategoryList) => {
+        try{
+            const response = await axios.get("/api/categories");
+            if(response?.data?.categories){
```
Comment (bhtibrewal) [inline src/Context/data.context.js:17]: Suggestion: instead of this you should do status check here

**Review by bhtibrewal — APPROVED** [review]
given few small suggestions, else everything is okay, good to merge

File: src/Components/Sidebar/Sidebar.jsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
+import { AiOutlineHome, AiOutlineLike, MdOutlineWatchLater, MdPlaylistPlay, RiHistoryFill} from "../../assets";
+
+export const Sidebar = () => {
+    return (
+        <aside className="aside-box">
+            <ul className="aside-items-box flex-column gap-1">
+                <li className="aside-items flex-row gap-1">
+                    <AiOutlineHome className="icon-vr-align mt-4-px"/> 
+                    <span className="noto-fonts">Home</span>
```
Comment (Ankur9669) [inline src/Components/Sidebar/Sidebar.jsx:9]: Suggestion: If Possible use p tags here for accesibility

**Review by Ankur9669 — APPROVED** [review]
Apart from the suggestions given by bhavika, i can't find anything worth mentioning, The coding conventions as well as the UI are looking great

**Pull request merged**